suite-native/biometrics/src/components/BiometricsIcons.tsx

```diff
@@ -0,0 +1,70 @@
+import { Platform } from 'react-native';
+
+import { IconName, Icon } from '@suite-common/icons';
+import { HStack, Box } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const ICON_SIZE_DEFAULT = 64;
+
+const iconWrapperStyle = prepareNativeStyle(
+    (utils, { wrapperSize, showShadow }: { wrapperSize: number; showShadow: boolean }) => ({
+        padding: 12,
+        borderRadius: utils.borders.radii.round,
+        backgroundColor: utils.colors.backgroundSurfaceElevation2,
+        color: utils.colors.iconPrimaryDefault,
+        width: wrapperSize,
+        height: wrapperSize,
+        shadowColor: utils.boxShadows.small.shadowColor,
+        shadowOffset: utils.boxShadows.small.shadowOffset,
+        shadowRadius: utils.boxShadows.small.shadowRadius,
+        elevation: utils.boxShadows.small.elevation,
+        shadowOpacity: utils.boxShadows.small.shadowOpacity,
+    }),
+);
+
+type BiometricsIconsProps = {
+    isFacialEnabled: boolean;
+    isFingerprintEnabled: boolean;
+    iconSize?: number;
+    showShadow?: boolean;
+};
+
+export const BiometricsIcons = ({
+    isFacialEnabled,
+    isFingerprintEnabled,
+    iconSize = ICON_SIZE_DEFAULT,
+    showShadow = false,
+}: BiometricsIconsProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const faceRecognitionIcon: IconName = Platform.OS === 'ios' ? 'faceId' : 'face';
+    const fingerprintIcon: IconName = Platform.OS === 'ios' ? 'touchId' : 'fingerprint';
+    const primaryIcon: IconName = isFingerprintEnabled ? fingerprintIcon : faceRecognitionIcon;
+
+    const iconWrapperSize = iconSize + 12 + 12;
```

Comment:
nit: Please add const for this number so it's obvious why this number specifically

Reply:
492763e1d2